Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -182,7 +182,9 @@ protected virtual void PointerIn()
             OnDestinationMarkerEnter(SetDestinationMarkerEvent(pointerContactDistance, pointerContactTarget, destinationPosition, controllerIndex));
 
             interactableObject = pointerContactTarget.GetComponent<VRTK_InteractableObject>();
-            if (interactableObject && interactableObject.pointerActivatesUseAction && interactableObject.holdButtonToUse)
+            bool cannot_use_because_not_grabbed = (interactableObject.useOnlyIfGrabbed && !interactableObject.IsGrabbed());
```

Comment:
also you need to define the access modifier. so this should be

`private bool <varname>`